mantidproject/mantid
# Vesuvio LoadVesuvio - LoadMonitor workspace uses full TOF range

The monitor workspaces optionally loaded by LoadVesuvio now load from 0 - 20000 in the TOF range

**To test:**

Run the following script:

```
loaded_data = LoadVesuvio(Filename='23271', SpectrumList='40-50', Mode='SingleDifference', LoadMonitors=True)
```

ensure that the loaded_data_monitor workspace has two spectra both of which have an X Range of 0 to 20000 (5-19990 for actually X values)

Fixes #16635

[RELEASE NOTES](https://github.com/mantidproject/mantid/blob/79f37e451581e7f2acfa58a5d1bc2f7d57a7a6c7/docs/source/release/v3.8.0/indirect_inelastic.rst#improvements)

---
#### Reviewer

Please comment on the following ([full description](http://www.mantidproject.org/Individual_Ticket_Testing)):
##### Code Review
##### Functional Tests
- Does everything look good? Comment with the ship it emoji but don't merge. A member of `@mantidproject/gatekeepers` will take care of it.
